corehq/apps/hqwebapp/static/hqwebapp/js/initial_page_data.js

```diff
@@ -28,15 +28,7 @@ hqDefine('hqwebapp/js/initial_page_data.js', function () {
                 if (COMMCAREHQ_INITIAL_PAGE_DATA[data.name] !== undefined) {
                     throw new Error("Duplicate key in initial page data: " + data.name);
                 }
-                var value = data.value;
-
-                // The initial_page_data tag escapes whitespace with \n, etc. for the sake of
-                // printing the value in a data attribute. Unescape here.
-                if (typeof value === "string") {
-                    value = JSON.parse('"' + value + '"');
```

Comment:
I was wrong when I wrote this, so the comment here was wrong; it's okay to have multi-line values in html attributes.